srt-protocol/src/protocol/sender/mod.rs

```diff
@@ -427,10 +432,14 @@ impl Sender {
 
     fn on_snd_event(&mut self, now: Instant) {
         self.snd_timer.reset(now);
-        self.send_control(KeepAlive, now);
         self.step = SenderAlgorithmStep::Step1;
     }
 
+    fn on_keepalive_event(&mut self, now: Instant) {
+        self.keepalive_timer.reset(now);
```

Comment:
@russelltg the keepalive timer is reset here

Reply:
Right but we should also reset it when we send a packet